openmc/element.py

```diff
@@ -45,9 +45,9 @@ def __init__(self, name='', xs=None):
 
     def __eq__(self, other):
         if isinstance(other, Element):
-            if self._name != other._name:
+            if self._name != other.name:
```

Comment:
How about using the `name` property directly with `self` as well, a in `self.name`?